examples/simple_module/client.py

```diff
@@ -7,9 +7,9 @@
 
 
 async def connect():
-    creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
-    opts = RobotClient.Options(refresh_interval=0, dial_options=DialOptions(credentials=creds), log_level=logging.DEBUG)
-    return await RobotClient.at_address("<your robot uri here>", opts)
+    # creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
```

Comment:
assuming you meant to uncomment

```suggestion
    creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
```

Reply:
nvm i see we are just changing to insecure

Reply:
Might have to un-import `Credentials` now that we're not using it anymore.

Reply:
ah that's unintentional, will revert that part